src/components/AccountDrawer/MiniPortfolio/Activity/ActivityRow.tsx

```diff
@@ -52,7 +53,7 @@ export function ActivityRow({
         descriptor={
           <ActivityRowDescriptor color="textSecondary">
             {descriptor}
-            {ENSName ?? otherAccount}
+            {ENSName ?? shortenAddress(otherAccount ?? '')}
```

Comment:
non-blocking nit: `shortenAddress` could accept an undefined address and return the empty string